.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+<!--- *This report is only for reporting bugs* -->
+
+---
+#### BUG REPORT
+
+name: "title of issue"\
+about: "short description of the issue"
+
+---
```

Comment:
I don't think these templates are valid. Can you make a demo repo?

Reply:
You are right, creating this the legacy way, isn't working as expected - I remember it used to work. Might be useful to create [https://stackoverflow.com/a/61580169](https://stackoverflow.com/a/61580169) via issue template builder.

Reply:
I created 2 templates via "issue template builder" today, and they get picked up by the GitHub, where adding it manually https://github.com/lubej/explorer/tree/master/.github/ISSUE_TEMPLATE, they don't get picked up. So I suggest, that the templates in this PR, are just copied to the issue template builder.

Reply:
This is probably because the templates don't have a valid syntax.

From https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository:
> issue templates must be located in the `.github/ISSUE_TEMPLATE` folder and contain valid `name:` and `about:` keys in the YAML frontmatter (for issue templates defined in .md files) or valid `name:` and `description:` keys (for issue forms defined in .yml files).

Reply:
Right, overlooked that. Legacy way was a bit different. Thanks for pointing that out.